projects/bp-gallery/src/components/common/picture-gallery/HorizontalPictureGrid.tsx

```diff
@@ -0,0 +1,375 @@
+import { Portal } from '@mui/material';
+import { Dispatch, SetStateAction, useCallback, useEffect, useMemo, useRef, useState } from 'react';
+import ScrollBar from 'react-perfect-scrollbar';
+import { useSimplifiedQueryResponseData } from '../../../graphql/queryUtils';
+import { root } from '../../../helpers/app-helpers';
+import { pushHistoryWithoutRouter } from '../../../helpers/history';
+import useGetPictures, { TextFilter } from '../../../hooks/get-pictures.hook';
+import { FlatPicture, PictureOverviewType } from '../../../types/additionalFlatTypes';
+import PictureView from '../../views/picture/PictureView';
+import PicturePreview from './PicturePreview';
+import { zoomIntoPicture, zoomOutOfPicture } from './helpers/picture-animations';
+
+const IMAGES_PER_WIDGET = 3;
+const IMAGE_WIDGET_WIDTH = 270;
+const TWO_HOURS = 2 * 60 * 60 * 1000;
+
+const SinglePicture = ({
+  picture,
+  size,
+  allowClicks,
+  navigateToPicture,
+  loading,
+}: {
+  picture?: FlatPicture;
+  size: string;
+  allowClicks?: boolean;
+  navigateToPicture: (id: string) => void;
+  loading: boolean;
+}) => {
+  return (
+    <div className={`${size === 'small' ? 'w-[125px] h-[100px]' : 'w-[260px] h-[200px]'}`}>
+      {picture ? (
+        <PicturePreview
+          picture={picture}
+          adornments={[]}
+          allowClicks={allowClicks}
+          onClick={() => {
+            if (!allowClicks) return;
+            navigateToPicture(picture.id);
+          }}
+          inverse={true}
+        />
+      ) : (
+        <div
+          className='horizontal-picture-placeholder w-full h-full'
+          style={{ flex: `1 1 0`, visibility: loading ? 'visible' : 'hidden' }}
+        />
+      )}
+    </div>
+  );
+};
+
+const PicturesWidget = ({
+  reverse,
```

Comment:
```suggestion
  reverse = false,
```
and then remove the `?? false` below